nest/src/progressbar.rs

```diff
@@ -39,27 +39,32 @@ impl ProgressBar {
     /// colors, and cannot go over 8 chars.
     /// Maximum value is 100.
     pub fn new(action: String) -> ProgressBar {
+        let refresh_rate = Duration::new(0, NANOS_PER_SEC / 10);
```

Comment:
Extracting `NANOS_PER_SEC` isn't à bad idea, it would probably make sense for the duration to be a constant or a parameter rather than a local variable (I may be mistaken, because you know, rust)

Reply:
You mean put the whole `Duration::new(...)` as a constant variable instead of a local one?

Reply:
Yes, or if not possible, the `NANOS_PER_SEC / 10`.

Reply:
Originally, i extracted the `NANOS_PER_SEC` not because it was a constant but because it looks like a magic number (to name it),  but i guess i can move the whole block up in a static const variable.